tests/test_register.py

```diff
@@ -0,0 +1,26 @@
+def test_register():
+    from archinfo.arch import Register
+
+    register = Register( name = 'MIPS', size = 8, vex_offset = 10,
+                         vex_name = 'xyz', subregisters = ['a', 'b'],
+                         alias_names = ('r0'), general_purpose = True, floating_point = True,
+                         vector = True, argument = True, persistent = True,
+                         default_value = (True, 'global'), linux_entry_value = 'argv',
+                         concretize_unique = True, concrete = True, artificial = True )
+    assert register.name == 'MIPS'
+    assert register.size == 8 
+    assert register.vex_offset == 10
+    assert register.vex_name == "xyz"
+    assert register.subregisters == ['a', 'b']
+    assert register.alias_names == ('r0')
+    assert register.general_purpose == True
```

Comment:
All these True asserts can be simplified to just test the value, this is why the CI is failing.